hera_corr_cm/hera_corr_cm.py

```diff
@@ -253,36 +253,54 @@ def get_f_status(self):
-            uptime (int)     : Multiples of 500e6 ADC clocks since last programming cycle
+            is_programmed (bool): True if the host is programmed
+            adc_is_configured (bool): True if the host adc is configured
+            is_initialized (bool): True if host is initialized
+            dest_is_configured (bool): True if dest_is_configured
+            version (str)      : Version of firmware installed
+            sample_rate (float): Sample rate in MHz
+            input (str)        : comma-delimited list of stream inputs, e.g. adc,adc,adc,adc,adc,adc
+            pmb_alert (bool)   : True if SNAP PSU controllers have issued an alert. False otherwise.
+            pps_count (int)    : Number of PPS pulses received since last programming cycle
+            serial (str)       : Serial number of this SNAP board
+            temp (float)       : Reported FPGA temperature (degrees C)
+            uptime (int)       : Multiples of 500e6 ADC clocks since last programming cycle
             last_programmed (datetime) : Last time this FPGA was programmed
             timestamp (datetime) : Asynchronous timestamp that these status entries were gathered
 
             Unknown values return the string "None"
         """
         stats = self._get_status_keys("snap")
-        conv_methods = {
-            'pmb_alert': lambda x: bool(int(x)),
-            'pps_count': int,
-            'serial': str,
-            'temp': float,
-            'uptime': int,
-            'last_programmed': dateutil.parser.parse,
-            'timestamp': dateutil.parser.parse,
+        conv_info = {
+            'is_programmed': ('is_programmed', lambda x: (x == 'True')),
+            'adc_is_configured': ('adc_is_configured', lambda x: (x == '1')),
+            'is_initialized': ('is_initialized', lambda x: (x == '1')),
+            'dest_is_configured': ('dest_is_configured', lambda x: (x == '1')),
+            'version': ('version', str),
+            'sample_rate': ('sample_rate', float),
+            'input': ('input', str),
+            'pmb_alert': ('pmb_alert', lambda x: bool(int(x))),
+            'pps_count': ('pps_count', int),
+            'serial': ('serial', str),
+            'temp': ('temp', float),
+            'uptime': ('uptime', int),
+            'last_programmed': ('last_programmed', dateutil.parser.parse),
+            'timestamp': ('timestamp', dateutil.parser.parse),
         }
-        rv = {}
+        f_status = {}
         for host, val in stats.items():
-            rv[host] = {}
-            for key, convfunc in conv_methods.items():
+            f_status[host] = {}
+            for key, (ckey, cfunc) in conv_info.items():
                 try:
-                    rv[host][key] = convfunc(stats[host][key])
-                except:
-                    rv[host][key] = "None"
-        return rv
+                    f_status[host][key] = cfunc(stats[host][ckey])
+                except:  # noqa
```

Comment:
bare excepts aren't great...

Reply:
I don't know enough to specify exceptions -- I'll make an issue.